draft-west-cookie-incrementalism.md

```diff
@@ -188,9 +188,41 @@ to:
 
 ~~~
 13. If the cookie-attribute-list contains an attribute
-    with an attribute-name of "SameSite", set the
+    with an attribute-name of "SameSite" and an
+    attribute-value of "Strict", "Lax", or "None", set the
     cookie's same-site-flag to attribute-value. Otherwise,
-    set the cookie's same-site-flag to "Lax".
+    set the cookie's same-site-flag to "Unset".
+~~~
+
+And finally by altering the fifth bullet point of step 1 of the cookie-string construction algorithm
+in Section 5.5 of {{RFC6265bis}} from:
+
+~~~
+ *  If the cookie's same-site-flag is not "None", and the HTTP
+    request is cross-site (as defined in Section 5.2) then exclude
+    the cookie unless all of the following statements hold:
+
+    1.  The same-site-flag is "Lax"
+
+    2.  The HTTP request's method is "safe".
+
+    3.  The HTTP request's target browsing context is a top-level
+        browsing context.
+~~~
+
+to:
+
+~~~
+ *  If the cookie's same-site-flag is not "None", and the HTTP
+    request is cross-site (as defined in Section 5.2) then exclude
+    the cookie unless all of the following statements hold:
+
+    1.  The same-site-flag is "Lax" or "Unset".
```

Comment:
"Default" would make sense here as well, I think, and be more clear than "Unset".

Reply:
Changed to "Default.